hw/mcu/dialog/da1469x/syscfg.yml

```diff
@@ -45,6 +45,19 @@ syscfg.defs:
             Specifies settling time [us] of 32M crystal oscillator.
         value: 2000
 
+    MCU_CLOCK_RCX_SWITCH:
+        description: Enable if switching to RCX
+        value: 0
+
+    MCU_CLOCK_RCX_SETTLE_TIME_MS:
+        description: >
+          Specifies settling time [ms] of RCX.
+        value: 1000
```

Comment:
Is the settling time of this clock really 1 second or is it 1ms? This could be a copy and paste error from the 32KHz xtal code (as that says 8000 ms for settling time and 8 seconds seems very long).